src/server/services/generation/generation.service.ts

```diff
@@ -349,6 +352,26 @@ export async function refreshGenerationCoverage() {
         "serviceProviders" = EXCLUDED."serviceProviders";
   `);
 
+  const updatedModels = await dbWrite.modelVersion.findMany({
```

Comment:
I guess we can use `dbRead` here instead since we are not doing mutations to the db
```suggestion
  const updatedModels = await dbRead.modelVersion.findMany({
```